client/src/components/PredictionPanel.vue

```diff
@@ -1,39 +1,88 @@
 <template>
   <div class='container'>
-    <map-geocoder-wrapper v-if='showSearch'
-                          v-model:expandSearch='expandSearch' />
-    <div class='lead-prediction' v-if='showPrediction'>
-      You have a high likelihood of lead.
+    <div class='explain-text'>
+      {{ ScorecardSummaryMessages.LEADOUT }}
+    </div>
+    <map-geocoder-wrapper v-model:expandSearch='showSearch' />
+    <div class='h1-header semi-bold'
+         v-if='this.pwsId != null && percentLead != null'>
+      Your neighborhood water system {{ pwsId }} is likely
+      {{ percentLead }}% lead.
+    </div>
+    <div class='explain-text'>
+      {{ ScorecardSummaryMessages.LEAD_LIKELIHOOD_EXPLAINED }}
     </div>
   </div>
 </template>
 
 <script lang='ts'>
 import { defineComponent } from 'vue';
 import MapGeocoderWrapper from './MapGeocoderWrapper.vue';
+import { dispatch, useSelector } from '../model/store';
+import { ScorecardSummaryMessages } from '../assets/messages/scorecard_summary_messages';
+import { getWaterSystem } from '../model/slices/lead_data_slice';
+import { GeoState } from '../model/states/geo_state';
+import { LeadDataState } from '../model/states/lead_data_state';
 
 /**
  * Container lead prediction.
  */
 export default defineComponent({
   name: 'PredictionPanel',
   components: { MapGeocoderWrapper },
+  setup() {
+    // Listen to state updates.
+    const geoState = useSelector((state) => state.geos) as GeoState;
+    const leadDataState = useSelector((state) => state.leadData) as LeadDataState;
+
+    return {
+      geoState: geoState,
+      leadState: leadDataState,
+    };
+  },
   data() {
     return {
       expandSearch: true,
       showSearch: true,
-      showPrediction: false,
+      ScorecardSummaryMessages,
     };
   },
+  computed: {
+    // Predicted estimate of lead.
+    percentLead(): number | null {
+      const leadServiceLines = this.leadState?.data?.leadServiceLines;
+      const serviceLines = this.leadState?.data?.serviceLines;
+
+      // Protect against dividing by 0
+      if (leadServiceLines != null && serviceLines != null && serviceLines != 0) {
+        return Math.round(leadServiceLines / serviceLines);
+      }
+      return null;
+    },
+    pwsId(): string | null {
+      return this.geoState?.geoids?.pwsId ?? null;
```

Comment:
same here - should we indicate that there is no pwsId in the message? fine to leave a TODO for both

Reply:
Same here